frontend/src/common/components/GeoJsonMap/index.js

```diff
@@ -0,0 +1,210 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import mapboxgl from 'mapbox-gl';
+
+import {
+    LoadingAnimation,
+} from '../../../public/components/View';
+
+const propTypes = {
+    className: PropTypes.string,
+    onAreaClick: PropTypes.func,
+    selections: PropTypes.arrayOf(PropTypes.string),
+    geoJson: PropTypes.object,  // eslint-disable-line
+    geoJsonBounds: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.number)),
+};
+
+const defaultProps = {
+    className: '',
+    onAreaClick: undefined,
+    selections: [],
+    geoJson: undefined,
+    geoJsonBounds: undefined,
+};
+
+
+export default class GeoJsonMap extends React.PureComponent {
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            map: undefined,
+        };
+    }
+
+    componentDidMount() {
+        this.mounted = true;
+
+        mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_ACCESS_TOKEN;
+        const map = new mapboxgl.Map({
+            center: [50, 10],
+            container: this.mapContainer,
+            style: process.env.REACT_APP_MAPBOX_STYLE,
+            zoom: 2,
+        });
+
+        map.on('load', () => {
+            if (this.mounted) {
```

Comment:
Isn't this.mounted used here unncessarily? Doesn't this only suppress the error instead of addressing it anyway?